grazer/src/main/java/edu/asu/diging/grazer/core/conceptpower/impl/ConceptpowerConnector.java

```diff
@@ -62,4 +65,21 @@ public IConcept getConcept(String id) {
         }
         return null;
     }
+    
+    @Override
+    @Cacheable(value = "concepts")
+    public ConceptpowerConcepts search(String item, String pos) {
+        
+        HttpHeaders requestHeaders = new HttpHeaders();
+        requestHeaders.setAccept(
+                Arrays.asList(new MediaType[] { MediaType.APPLICATION_JSON }));
+        HttpEntity<?> entity = new HttpEntity<Object>(requestHeaders);
+        
+        ResponseEntity<ConceptpowerConcepts> response = restTemplate.exchange(
+                conceptpowerUrl + searchEndpoint + item + "/" + pos, 
+                HttpMethod.GET, entity, ConceptpowerConcepts.class);
+        
+        return response.getBody();
```

Comment:
you need to make sure only concepts that are actually used in one of the statements are returned